src/NPCGeneration/createNPC.ts

```diff
@@ -57,7 +57,10 @@ export const createNPC = (town: Town, base = defaultBase): NPC => {
   const profession = base.profession || lib.fetchProfessionChance(town, base as NPC)
 
   const firstName = base.firstName || getFirstName(race, base.gender)
-  const lastName = base.lastName || getLastName(race)
+  let lastName = base.lastName || getLastName(race)
+  if (lastName === firstName) {
+    lastName = firstName
```

Comment:
This assignment does nothing. Why is this here?

Reply:
It should be is lastName === '' then lastname = firstname.

Reply:
Just edited your comment to make it clearer :)